pages/configuration/elasticsearch.rst

```diff
@@ -338,7 +338,7 @@ Save the following index template for the custom index mapping into a file named
 
 Finally, load the index mapping into Elasticsearch with the following command::
 
-  $ curl -X PUT -d @'graylog-custom-mapping.json' 'http://localhost:9200/_template/graylog-custom-mapping?pretty'
+  $ curl -X PUT -d @'graylog-custom-mapping.json' -H 'Content-Type: application/json' 'http://localhost:9200/_template/graylog-custom-mapping?pretty'
```

Comment:
thank you for this catch. 

Internally we moved to using [httpie](https://httpie.org/) that is why such missing bits are unseen.